gatsby/pages.rst

```diff
@@ -16,64 +24,62 @@ If we see the file structure of our `hello-world` project, we can see that there
     │       └── page-2.js
     ...
 
-Let's see how is made `index.js` file. This page is the homepage of our example site.
+Let's see how is made ``index.js`` file. This page is the homepage of our example site.
```

Comment:
"Let us see how the ``index.js`` file is made."

One line per sentence.

"home page" is two words.